### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by do-borisov** [jsGanttImproved/jsgantt-improved#143]
Hi. Trying to make something custom for my needs. It is similar to what you have achieved via the pDependable in a way... What I want to achieve is a single-row-object with multiple pDependable-arrows refferencing between eachother ... Your demo has something very close to what I need with the objects "Task Label Table" & "Task Scrolling Grid" and the arrow showing the time-frame between them... I'm  adding all the objects via the g.AddTaskItemObject manually for my needs,

**Comment by mariohmol** [comment]
Hi.. did u find a solution for this?

**Comment by do-borisov** [comment]
Hi @mariohmol. Kind of, yes... I wasn't sure if it's up to me or the lib functionality. In my case it was more of my side I guess. Since the sub-objects I wanted to represent were not 'actual-objects', but more of a visual-representation of a time-gap of the object.. And it got quite messy with the id's(pDepend) of each sub-objects ... I guess just fyi I got end_date not pointing to the correct calendar-date.. But could be and overloaded with data, or as I saw and other people got some issues when the dates are extended over 5-6 years and above.
